src/test/memory/test_mmap.rs

```diff
@@ -181,6 +187,48 @@ fn test_mmap_anon() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+fn test_munmap_front() -> Result<(), Box<dyn Error>> {
```

Comment:
A short comment for each test would be helpful to explain what's being tested or why.

Reply:
Done